lib/github_changelog_generator/fetcher.rb

```diff
@@ -123,7 +124,11 @@ def fetch_closed_issues_and_pr
     def fetch_closed_pull_requests
       pull_requests = []
       begin
-        response = @github.pull_requests.list @options[:user], @options[:project], state: "closed"
+        if @options[:release_branch].nil?
+          response = @github.pull_requests.list @options[:user], @options[:project], state: "closed"
+        else
+          response = @github.pull_requests.list @options[:user], @options[:project], state: "closed", base: @options[:release_branch]
+        end
```

Comment:
I can't help but think there would a be a better way? The core of fetch_closed_pull_requests is this call, that is the sole purpose of it, so wrapping it in another function seems superfluous.
I have an idea give me another day to play.